### PR TITLE
Documented login screen and added pictures

### PR DESCRIPTION
**Pull request by Sworgon**
The overall purpose of documenting the program is to describe the overall structure of the program and not specifically code.

For solving this issue you can do the following:

Relations:
- Write about which BLoCs/Screens this class is related to.

Responsibility/Functionality:
- Write about the responsibility/functionality of the current class and how it interacts with it’s related BLoCs/Screens (e.g. this stream is used to send weekplans to this screen, so they can be displayed).

Note: If you have any suggestions to what we can change, do not hesitate to contact us :D

### CODE REVIEW

**Comment by User12837985123** [comment]
@aau-giraf/sw610f19 you have been assigned this. Here's your checklist. I marked @Moesgaarda as the reviewer, but you might change it as you like.
Please use the following checklist:
- [x] Is there documented which BLoCs/Screens the current class is related to?

- [x] Is the responsibility/functionality of the current class explained? Also regarding the classes it is related to?

- [x] If any unusual decisions are introduced in the code are they reasoned about in the documentation?

Note: If you have any suggestions to what we can change, do not hesitate to contact us :D

File: file_documentation/weekplanner/screens/login_screen.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+In the center of the screen is the Giraf icon.
+
+The green square marks the username text field. The user is supposed to write their username in this text field.
+
+The blue square marks the password text field. The user is supposed to write their password in this text field. Whatever is written in this field is hidden, to keep the password hidden from other people.
+
+The red square marks the login button.
+
+The purple square marks the auto-login button.
```
Comment (Tekrus) [inline file_documentation/weekplanner/screens/login_screen.md]:
```suggestion
The purple square marks the auto-login button and is only visible in debug mode.
```

File: file_documentation/weekplanner/screens/login_screen.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Button
```
Comment (Tekrus) [inline file_documentation/weekplanner/screens/login_screen.md:20]: Auto-login button?

**Review by Tekrus — CHANGES_REQUESTED** [review]

File: file_documentation/weekplanner/screens/login_screen.md (hunk 2 above)
Comment (Sworgon) [reply file_documentation/weekplanner/screens/login_screen.md:20]: yes

**Review by Tekrus — APPROVED** [review]

**Pull request merged**